thirtyinch/src/main/java/net/grandcentrix/thirtyinch/TiActivity.java

```diff
@@ -121,7 +121,7 @@ protected void onStop() {
 
     @CallSuper
     @Override
-    protected void onSaveInstanceState(final Bundle outState) {
+    protected void onSaveInstanceState(@NonNull final Bundle outState) {
```

Comment:
There is no nullability annotation on this parameter in the appcompat library. Are you sure it's `@NonNull`?

Reply:
There is in version 1.1.0